lib/cppcheck.cpp

```diff
@@ -1724,6 +1724,7 @@ void CppCheck::getErrorMessages(ErrorLogger &errorlogger)
         (*it)->getErrorMessages(&errorlogger, &s);
 
     CheckUnusedFunctions::getErrorMessages(errorlogger);
+    s.addEnabled("missingInclude");
```

Comment:
hmm I wonder why we don't just write `s.addEnabled("all");` at line 1710. we don't need to enable those severities explicitly then. do you see some problems when doing that?

Reply:
That looks cleaner, I'll try that.